src/properties/misc/optionproperty.cpp

```diff
@@ -207,26 +207,13 @@ std::string OptionProperty::stringValue() const {
 }
 
 std::string OptionProperty::generateAdditionalJsonDescription() const {
-    // @REFACTOR from selectionproperty.cpp, possible refactoring? ---abock
-    std::string result = "{ \"" + OptionsKey + "\": [";
-    for (size_t i = 0; i < _options.size(); i++) {
-        const Option& o = _options[i];
-        const std::string v = std::to_string(o.value);
-        const std::string vSan = escapedJson(v);
-        const std::string d = o.description;
-        const std::string dSan = escapedJson(d);
-
-        result += '{';
-        result += std::format(R"("{}": "{}")", vSan, dSan);
-        result += '}';
-
-        if (i != _options.size() - 1) {
-            result += ",";
-        }
+    nlohmann::json data = {};
```

Comment:
```suggestion
    nlohmann::json data = nlohmann::object();
```

?

Reply:
I think the empty curly braces will produce maybe undefined on the frontend side which is better because we can check for undefined easier than for empty objects

Reply:
Ah.. In that case `nlohmann:json data;` will represent that intention better